src/parse/parser.rs

```diff
@@ -864,6 +864,8 @@ impl<'help, 'app> Parser<'help, 'app> {
             }
         }
 
+        let partial_parsing_enabled = self.is_set(AS::IgnoreErrors);
```

Comment:
You don't need this line here.

Reply:
Later where I need to check the flag, "self" is alread borrowed if I inline it:

```
error[E0502]: cannot borrow `*self` as immutable because it is also borrowed as mutable
   --> src/parse/parser.rs:916:24
    |
867 |         if let Some(sc) = self.app.subcommands.iter_mut().find(|s| s.name == sc_name) {
    |                           -------------------- mutable borrow occurs here
...
916 |                     if self.is_set(AS::IgnoreErrors) {
    |                        ^^^^ immutable borrow occurs here
...
926 |             let name = sc.name.clone();
    |                        ------- mutable borrow later used here

error: aborting due to previous error
```
What do you suggest?